modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -1297,6 +1298,44 @@ Supported Values:
 `groupID:assetID:version,groupID2:assetID:version` |
 |===
 
+=== exchange asset mutableDataUpload
+
+----
+> exchange asset mutableDataUpload [properties] <assetIdentifier>
+----
+
+This command modify mutable data (tags, categories, fields and documentation) to an already created asset. 
+
+Use the following format for the _assetIdentifier_ argument:
+
+`([_groupID_]/)_assetID_/_version_`
```

Comment:
😂  I had also done it in this way, but we are not using this format for optional arguments. Check this

https://github.com/mulesoft/docs-runtime-manager/pull/521#discussion_r668241713

```suggestion
`[_<groupID>_]/_<assetID>_/_<version>_`
```